httplug/tutorial.rst

```diff
@@ -119,8 +119,8 @@ Using the promise directly
 If you don't want to use the callback system, you can also get the state of the promise with ``$promise->getState()``
 will return of one ``Promise::PENDING``, ``Promise::FULFILLED`` or ``Promise::REJECTED``.
 
-Then you can get the response of the promise if it's in ``FULFILLED`` state with ``$promise->getResponse()`` call or
-get the error of the promise if it's in ``REJECTED`` state with ``$promise->getRequest()`` call
+Then you can get the response of the promise if it's in ``FULFILLED`` state or get the error of the promise if it's in
```

Comment:
do we return the exception or throw it? i would say exception not error, and if we throw say trigger instead of get